packages/nest/src/oauth/application/oauth2.user.model.ts

```diff
@@ -76,30 +81,38 @@ export class OAuth2UserModel extends OAuth2BaseModel implements PasswordModel, R
                 token.scope,
                 user,
             );
+
             if (callback) {
                 callback(null, final);
             }
+
             return final;
         } catch (err) {
             if (callback) {
                 callback(new ForbiddenException('Invalid refresh token'), undefined);
             }
+
             return null;
         }
     }
 
-    async revokeToken(token: RefreshToken | Token, callback?: (err?: any, result?: boolean) => void): Promise<boolean> {
+    public async revokeToken(token: RefreshToken | Token, callback?: Callback<boolean>): Promise<boolean> {
         let accessToken;
+
         if ('accessToken' in token) {
             accessToken = token['accessToken'];
         }
+
         const refreshToken = token.refreshToken;
         await this.deleteTokenInteractor.execute(accessToken, refreshToken);
+
         if (callback) {
             callback(null, true);
         }
+
         return true;
     }
+
     /*
```

Comment:
Clean up

Reply:
Removed in 11def0af7d6000420095a054acb90e3be4e29422